webplatformcompat/tests/test_view_serializers.py

```diff
@@ -1,37 +1,47 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""
-Tests for web-platform-compat.view_serializers.
-
-TODO: Refactor this code to be more unit tests than integration tests.
-This should wait until bug 1153288 (Reimplement DRF serializers and renderer)
-"""
+"""Tests for view serializers."""
 from __future__ import unicode_literals
-from json import dumps, loads
 
 from django.test.utils import override_settings
+from drf_cached_instances.models import CachedQueryset
+from rest_framework.test import APIRequestFactory
```

Comment:
For the record, every time I see `*RequestFactory` I think of `C&C Music Factory`. I don't know why.